omui/src/components/icon/icon.stories.js

```diff
@@ -2,21 +2,15 @@ import React from 'react';
 import { withKnobs, select } from '@storybook/addon-knobs';
 import { FaBeer } from 'react-icons/fa';
 
-import { Icon, IconStyle } from './';
+import { Icon } from './';
 
-import {
-  themeIconSizes,
-  themeAllPossibleColors
-} from '../../helpers/get-theme';
+import { themeAllPossibleColors } from '../../helpers/get-theme';
 
 export default { title: 'Components/Icon', decorators: [withKnobs] };
 
-const { size } = IconStyle.defaultProps;
-
 export const Default = () => (
   <Icon
     as={FaBeer}
```

Comment:
Why no size? We should allow for this prop to be changed in the story...

Reply:
The `IconStyle` was removed on chakra. A new numeric input (in pixels) was added to icon stories.